src/PaymentElement.res

```diff
@@ -17,17 +17,17 @@ let make = (~cardProps, ~expiryProps, ~cvcProps, ~paymentType: CardThemeType.mod
     layout,
     customerPaymentMethods,
     displaySavedPaymentMethods,
+    sdkHandleConfirmPayment,
   } = Recoil.useRecoilValueFromAtom(optionAtom)
+  let logger = HyperLogger.make(~source=Elements(Payment))
```

Comment:
Please use the logger from Recoil, instead of making a new one

Reply:
Updated